src/main/java/application/rest/OrderService.java

```diff
@@ -165,7 +185,28 @@ public Response getOrders() throws Exception {
             summary = "Create an order",
             description = "Uses RabbitMQ to notify a new shipping order."
     )
-    public Response create(Order payload, @Context UriInfo uriInfo) throws IOException, TimeoutException {
+    @Timed(name = "createOrders.timer",
+            absolute = true,
+            displayName = "createOrder Timer",
+            description = "Time taken to create an order.",
+            reusable = true)
+    @Counted(name = "createOrders",
+            absolute = true,
+            displayName = "createOrders call count",
+            description = "Number of times we've called createOrders.",
+            monotonic = true,
+            reusable = true)
+    @Metered(name = "createOrdersMeter",
+            displayName = "Orders Call Frequency",
+            description = "Rate the throughput of createOrders",
+            reusable = true)
```

Comment:
`reusable` is not required here for all the three annotations since we are not reusing the orders annotations in any other microservices